src/VCRCleanerEventSubscriber.php

```diff
@@ -33,6 +34,23 @@ public function onBeforeRecord(BeforeRecordEvent $event)
         $this->sanitizeRequestUrl($event->getRequest());
         $this->sanitizeRequestHeaders($event->getRequest());
         $this->sanitizeRequestBody($event->getRequest());
+
+        $orgResponse = $event->getResponse();
+        $modResponse = clone $event->getResponse();
```

Comment:
Instead of cloning here, I should just call `toArray()` since that's what both `sanitizeResponseHeaders` and `sanitizeResponseBody` are doing on their own. Then just pass that array by reference. No need to be redundant here.

After both those sanitize functions are called, call `Response::fromArray()` to prepare for reflection.